README.md

```diff
@@ -1 +1,103 @@
 # asl-public-api
+
+## About
+
+API server providing access to licence and user data
+
+## Usage
+
+To run a local instance:
+
+```
+npm run dev
+```
+
+## Dependencies
+
+* `@asl/service/api` provides the common api boilerplate - body-parsing, auth etc.
+* `@asl/schema` provides models for interacting with database objects
+
+## Configuration
+
+The service can be configured for local development by setting environment variables in a `.env` file.
+
+The following environment variables are required:
+
+* `WORKFLOW_SERVICE` - the url of the `asl-workflow` instance
+* `PERMISSIONS_SERVICE` - the url of the `asl-permissions` instance
+* `DATABASE_NAME` - the name of your postgres database
+* `KEYCLOAK_REALM` - the keycloak realm used for authentication
+* `KEYCLOAK_URL` - the url of the keycloak server
+* `KEYCLOAK_CLIENT` - the client name used to authenticate with keycloak
+* `KEYCLOAK_SECRET` - the secret used to authenticate with the keycloak client
+
+The following environment variables can be optionally defined:
+
+* `PORT` - port that the service will listen on - default `8080`
+* `DATABASE_HOST` - hostname of the postgres instance - default `localhost`
+* `DATABASE_PORT` - port of the postgres instance - default `5432`
+* `DATABASE_USERNAME` - username of the postgres instance - default `undefined`
+* `DATABASE_PASSWORD` - password of the postgres instance - default `undefined`
+
+## Connected services
+
+### Upstream
+
+* `asl` - establishment facing ui
+* `asl-inspector-ui` - internal facing ui.
+
+### Downstream
+
+The following services must be available in order to run:
+
+* `asl-permissions` - permissions microservice
+* `asl-workflow` - api providing workflow management on change requests
+* `postgres` - to store licence data
+
+## Development
+
+### Database setup
+
+Scripts for setting up a local database with dev data are available in the [`asl-schema` project](https://github.com/ukhomeoffice/asl-schema). First clone that repo and install the dependencies. Then run the following commands:
```

Comment:
Maybe explicitly state that the commands need to be run in the `asl-schema` dir rather than in `asl-public-api` to avoid any ambiguity?

Reply:
Think this is still outstanding?